x-pack/plugins/apm/public/components/app/correlations/use_failed_transactions_correlations.test.tsx

```diff
@@ -173,6 +173,8 @@ describe('useFailedTransactionsCorrelations', () => {
         jest.advanceTimersByTime(100);
         await waitFor(() => expect(result.current.progress.loaded).toBe(0));
         jest.advanceTimersByTime(100);
+        await waitFor(() => expect(result.current.progress.loaded).toBe(0));
```

Comment:
Not sure what this extra line is for... Is it just to increase the timer?

Reply:
Before this PR, the data for overall and failed transaction histogram were loaded in parallel. Now they are loaded consecutively because the failed transaction histogram relies on the min/max duration returned from the overall histogram request. Each request has a simulated duration of 100ms in these tests so I needed to add this additional `waitFor` to consider the additional request. I'll add some comments to explain the `waitFor`s!

Reply:
Added a comment in [4ad0a05](https://github.com/elastic/kibana/pull/138259/commits/4ad0a05e622756287a64e4f681bf9240855acbfc).